src/main/java/com/auth0/client/mgmt/RequestBuilder.java

```diff
@@ -16,8 +16,6 @@
     private final Auth0HttpClient client;
     private final TokenProvider tokenProvider;
     private final HttpMethod method;
-
-    // TODO decouple from OkHttp
```

Comment:
Discussed that it's fine for internals to use OkHttp for request/URL building. The `com.auth0.net.client` types are OkHttp-agnostic